.drone.yml

```diff
@@ -756,7 +756,7 @@ steps:
 
   # TODO: We should probably build all dependencies into a test image
   - name: test-e2e
-    image: mcr.microsoft.com/playwright:v1.32.1-focal
+    image: mcr.microsoft.com/playwright:v1.32.3-jammy
     commands:
       - curl -sLO https://go.dev/dl/go1.20.linux-amd64.tar.gz && tar -C /usr/local -xzf go1.20.linux-amd64.tar.gz
```

Comment:
Can you bump this version of go?

Reply:
Yeah, a shame golang does not provide stable download links to latest release, but we may be able to do some curl+js magic with https://go.dev/dl/?mode=json.

Reply:
Hmm, not as easy as I had hoped, the json only contains latest two supported versions, no structure to select only 1.20.x.

```bash
curl 'https://go.dev/dl/?mode=json' | fx
```

<img width="288" alt="image" src="https://user-images.githubusercontent.com/115237/233200174-adf678c1-936e-42e3-a898-e3e2b316b501.png">

Reply:
afaik it should always be the first result, though

Reply:
Yeah but i guess we don't always want latest.

Update to 1.20.3 in https://github.com/go-gitea/gitea/pull/24218/commits/6595fc907918ce25edfb0adf8dd080fe9d9e2ca7.

Reply:
Made it dynamic in https://github.com/go-gitea/gitea/pull/24218/commits/d5a30a7223a43cf7f86f2b320d94f03030764cb5.

It will now track the latest stable 1.20 version of go with this. I could also make it just always use the highest stable version, hmm.

Reply:
Done that, now tracking highest stable.